async-http-client/src/main/scala/org/http4s/asynchttpclient/AsyncHttpClient.scala

```diff
@@ -43,6 +43,7 @@ import org.reactivestreams.Publisher
 
 import scala.jdk.CollectionConverters._
 
+@deprecated("Upstream is unmaintained. Recommend choosing another backend.", "0.22.12")
```

Comment:
Also, a good question is will we support `AsyncHttpClient` within `1.0`? I tend to say yes gingerly. But let's then fix the deprecated-since version.
```suggestion
@deprecated("Upstream is unmaintained. Recommend choosing another backend.", "1.0")
```

Reply:
I'm an extremist, but honestly I say let's toss it. I love any excuse to slim down the build.

Reply:
Btw, I think we should say deprecated since 0.23.11. It was not deprecated in 1.0, it was deprecated _before_ 1.0 :)

Reply:
Very likely that download stats (of artifact) could help in the decision-making of removing. However, this removing is a real act of brave heart, you know :)

Reply:
It's would move to a new repo in the Great Schism anyway.

It's 9.4% of our client downloads in February.  On 1.0.0-M31, it's 4.0%.  Let's leave it in this milestone just to announce its demise to users who aren't watching every merge PR, and cut it in the next.

```scala
@deprecated("Upstream is unmaintained. This backend will be removed in the next milestone. If anyone wants to adopt it, please contact the http4s team.", "1.0.0-M32")
```